### ISSUE TITLE
Create a better name for Package Management page

### ISSUE THREAD

**Issue by ericonr** [void-linux/void-docs#381]
Following https://github.com/void-linux/void-docs/pull/368#discussion_r447384874 and @Duncaen's comments on IRC:

```
      duncaen │ https://docs.voidlinux.org/xbps/package-management.html
      duncaen │ this is a weird page
      duncaen │ this has a bunch of stuff about xbps, but its not really "Package management" imho
```

we should look into picking a better and more descriptive name for that section.

**Comment by flexibeast** [comment]
i feel this has been addressed by #384. Ok to close?

**Comment by ericonr** [comment]
Okdo! Thanks :D
